docs/preview/features/show-arcus-templates-in-visualstudio.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Show Arcus project templates in Visual Studio"
+layout: default
+---
+
+# Use Arcus project templates in Visual Studio
+
+The available .NET project templates in this repository are all available via the command line.
+For example:
+
+```shell
+PM > dotnet new --install Arcus.Templates.WebApi
+```
+
+Which will install the template on your machine so it can be used for creating new projects:
+
+```shell
+> dotnet new arcus-webapi --name Arcus.Demo.WebAPI
+```
+
+However, you can also use our .NET templates in Visual Studio by following these steps:
+
+1. Go to **Tools > Options** in Visual Studio
+2. Navigate to **Preview Features**
+3. Check the box **Show all .NET Core templates in the New project dialog**
+
+![Show all .NET Core templates in the New project dialog via Tools > Options > Preview Features](img/show-dotnet-templates-in-vs.png)
+
```

Comment:
We can hold of until we added an image here until the templates are actually released with the logo embedded. We can then add a snapshot when we finalize the docs to be published (outside preview).

Reply:
Can you install the template from our MyGet feed and install that one?

Reply:
Right now, I don't see any Arcus templates anymore; after I did an update 😄 .

Reply:
I think we can do a full release with all our improvements to be honest, then we can finalize this as well.